infra/terraform/README.md

```diff
@@ -0,0 +1,147 @@
+# OWTF Terraform Scripts
+
+## Overview of the Bash Script
+
+This script automates the process of modifying a Terraform configuration file, checking for unsafe modifications, and running Terraform commands. It also handles different operating systems (macOS and Linux) and allows the user to choose whether to run the script in a "safe mode" or "unsafe mode."
+
+## Deployment Steps
+
+1. **Clone the Repository**
+
+    First, clone the repository containing the deployment script and Kubernetes manifests:
+    ```bash
+    git clone https://github.com/owtf/owtf.git
+
+    cd owtf/infra/terraform
+    ```
+2. **Create AWS Access Keys**
+
+    Follow this [link](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey) and read the document to create access keys.
+3. **Install Terraform and AWS CLI**
+
+    Make sure to install [Terraform](https://developer.hashicorp.com/terraform/install) and [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+
+4. **Configure AWS CLI**
+
+    ```
+    aws configure
+    ```
+    Run this command to configure aws access key and secret access key of your aws iam user.
+
+    ```
+    AWS Access Key ID [None]: 
+    AWS Secret Access Key [None]: 
+    Default region name [None]: 
+    Default output format [None]: 
+    ```
+    It would prompt you to paste these details.
+5. **Define Variables**
+
+   ```bash
+   variables.tf
+   ```
+   This file holds the variables of the Terraform configuration that can be modified as user required.
+
+6. **Run the script**
+
+    ```bash
```

Comment:
instead of the script here, can you put this in the `Makefile`? something with targets to do `tf apply` and `tf plan`